app/upgrades/v4/old_rollapps.go

```diff
@@ -79,4 +79,36 @@ var (
 		"ibc/13B2C536BB057AC79D5616B8EA1B9540EC1F2170718CAFF6F0083C966FFFED0B", // OSMO
 		"ibc/04E01477A69DF1E5EE99F85C15B66D68D23292275357CAA44B2E0527310A405E", // EVMOS
 	}
+
+	rollappXRollappID        = "rollappx_700001-1"
+	rollappXInitialSupply, _ = math.NewIntFromString("2000060000000000000000000000")
+	rollappXGenesisInfo      = rollapptypes.GenesisInfo{
+		GenesisChecksum: "",
```

Comment:
need the genesis checksum here so we will wait till its populated in the relevant notion page (also for crynux)